### PR TITLE
Choice icon alignment

### PR DESCRIPTION
**Pull request by philschanely**
Adds a new setting for Choice (modifies Tabs as well as a result) that allows for radio and icon to be aligned to the top of the choice when desired.

Screen Shot 2021-03-17 at 11 27 58 AM

Also removes the `nowrap` setting that was present on Choice

### Testing and Impact

See new section on documentation page for Choice in docs site and new setting in React.

1. Adds new setting for aligning icons in Choice elements. No impact on current uses. Validate the following work as before:
   - [ ] Website > Navigation > + New in a menu > See Choice card in the Modal that appears
   - [ ] People > Import Contacts (new) > first page has Choice cards for selecting import modes

### CODE REVIEW

**Comment by pixelflips** [comment]
@philschanely was looking at this one but it looks like the base branch is `master` and not `develop` so there are lots of changes. Would you mind updating that?

File: packages/sage-assets/lib/stylesheets/patterns/elements/_choice.scss [hunk 1]
```diff
@@ -194,7 +213,7 @@ $-choice-radio-color-checked-inner: map-get($sage-radio-colors, checked-inner);
 .sage-choice__graphic {
   overflow: hidden;
   width: rem(48px);
-  height: remb(48px);
+  height: rem(48px);
```
Comment (pixelflips) [inline packages/sage-assets/lib/stylesheets/patterns/elements/_choice.scss:216]: Awesome eye!

**Review by pixelflips — APPROVED** [review]
👍🏼  LGTM! Nice work!

**Review by joedakroub — APPROVED** [review]
No concerns here. Nice addition Phil! 👍🏼

**Pull request merged**